app/Utils/dotNotationUtil.php

```diff
@@ -5,14 +5,14 @@ final class FreshRSS_dotNotation_Util
 {
 
 	/**
-	 * Get an item from an array using "dot" notation.
-	 * Functions adapted from https://stackoverflow.com/a/39118759
-	 * https://github.com/illuminate/support/blob/52e8f314b8043860b1c09e5c2c7e8cca94aafc7d/Arr.php#L270-L305
-	 * Newer version in
-	 * https://github.com/laravel/framework/blob/10.x/src/Illuminate/Collections/Arr.php#L302-L337
-	 *
-	 * @param \ArrayAccess<string,mixed>|array<string,mixed>|mixed $array
-	 */
+	* Get an item from an array using "dot" notation.
+	* Functions adapted from https://stackoverflow.com/a/39118759
+	* https://github.com/illuminate/support/blob/52e8f314b8043860b1c09e5c2c7e8cca94aafc7d/Arr.php#L270-L305
+	* Newer version in
+	* https://github.com/laravel/framework/blob/10.x/src/Illuminate/Collections/Arr.php#L302-L337
+	*
+	* @param \ArrayAccess<string,mixed>|array<string,mixed>|mixed $array
+	*/
```

Comment:
Wrong change of comment indenting and again, please do not perform unrelated edits